docs/testing/debug-initialize-onready.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Debug the initialize and onReady methods
+description: Learn how to debug the Office.initialize and Office.onReady methods.
+ms.date: 07/11/2022
+ms.localizationpriority: medium
+---
+
+# Debug the initialize and onReady methods
+
+> [!NOTE]
+> This article assumes that you are familiar with [Initialize your Office Add-in](../develop/initialize-add-in.md).
+
+The paradox of debugging the [Office.initialize](/javascript/api/office#Office_initialize_reason_) and [Office.onReady](/javascript/api/office#office-office-onready-function(1)) methods is that a debugger can only attach to a process that is running, but these methods run immediately as the add-in's runtime process starts up, before a debugger can attach. In most situations, restarting the add-in after a debugger is attached doesn't help because restarting the add-in closes the original runtime process *and the attached debugger* and starts a new process that has no debugger attached.
```

Comment:
The Office.initialize link opens to the top of the office package page. I think the tag for the Office.initialize section is '#office-office-initialize-function(1)'.

Reply:
```suggestion
The paradox of debugging the [Office.initialize](/javascript/api/office#office-office-initialize-function(1)) and [Office.onReady](/javascript/api/office#office-office-onready-function(1)) methods is that a debugger can only attach to a process that is running, but these methods run immediately as the add-in's runtime process starts up, before a debugger can attach. In most situations, restarting the add-in after a debugger is attached doesn't help because restarting the add-in closes the original runtime process *and the attached debugger* and starts a new process that has no debugger attached.
```